views/marts/product/fct_active_users.view.lkml

```diff
@@ -76,28 +76,84 @@ view: fct_active_users {
   dimension: daily_desktop_active_users {
     type: number
     sql: ${TABLE}.daily_desktop_active_users ;;
-    label: "Desktop/Client DAU"
-    description: "DAU reported by Rudderstack telemetry."
+    label: "Desktop DAU"
+    description: "Desktop DAU reported by telemetry."
   }
 
   measure: total_daily_desktop_active_users {
     type: sum
     sql: ${daily_desktop_active_users} ;;
-    label: "Total Desktop/Client DAU"
+    label: "Total Desktop DAU"
     view_label: "  * Metrics. DAU"
   }
 
-  dimension: daily_legacy_active_users {
+  dimension: daily_webapp_active_users {
     type: number
-    sql: ${TABLE}.daily_legacy_active_users ;;
-    label: "Legacy Desktop/Client DAU"
+    sql: ${TABLE}.daily_webapp_active_users ;;
+    label: "Webapp DAU"
+    description: "Webapp DAU reported by telemetry."
+  }
+
+  measure: total_daily_webapp_active_users {
+    type: sum
+    sql: ${daily_webapp_active_users} ;;
+    label: "Total Webapp DAU"
+    view_label: "  * Metrics. DAU"
+  }
+
+  dimension: daily_client_desktop_active_users {
+    type: number
+    sql: ${TABLE}.daily_client_desktop_active_users ;;
+    label: "Client Desktop DAU"
+    description: "Client DAU reported by Rudderstack telemetry."
+  }
+
+  measure: total_client_daily_desktop_active_users {
```

Comment:
`total_client_daily_desktop_active_users` -> `total_daily_client_desktop_active_users` to be consistent with below?